public/style.css

```diff
@@ -71,10 +71,47 @@ form label {
 .placard.raised {
     background-color: #5aaaff;
     border: 1px solid rgb(0, 108, 222);
-	color: #fff;
+    color: #fff;
     box-shadow: 0px 4px 16px rgba(0, 0, 0, 0.3);
 }
 
+.btn-primary {
+    background-color: #0071eb;
+    border-color: #0071eb;
+}
+
+.btn-primary.disabled,
+.btn-primary:disabled {
+    background-color: #eee;
+    border-color: #ddd;
+    color: #696969;
+    opacity: 1;
+}
+
+.placards {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
Are there mobile styles for the grid?

Reply:
This is good idea! I'll see if I can add this.

Reply:
I've just added responsive grid styles ☺️